woocommerce/woocommerce
# Update the copy for the terms and conditions on WooCommerce.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Based on internal discussion and recommendations from our legal team (reference: pdnLyh-5eI-p2#comment-3865), update the copy for the onboarding terms and conditions.

Closes #44435



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Start a new store while having this branch of Woo up and running. Alternatively, in case you already have a local install up and running, reset the onboarding flow to get access to the intro screen by installing the **WooCommerce Beta Tester** plugin and heading over to `Tools > WCA Test Helper > Tools > Reset Onboarding Wizard`
2. Click on WooCommerce > Home
3. When the following screen is displayed, make sure the copy matches the one exemplified on the screenshot below:

<img width="1231" alt="Screenshot 2024-02-08 at 08 13 44" src="https://github.com/woocommerce/woocommerce/assets/15730971/5fa297d1-42d5-49e9-93fd-7277e0e3c5ce">




### Changelog entry



-   [] Automatically create a changelog entry from the details below.

<details>

#### Significance




#### Type




#### Message 

Update the copy for the terms and conditions on WooCommerce.

#### Comment 

</details>
